tests/wpunit/ConnectionRegistrationTest.php

```diff
@@ -243,7 +243,7 @@ static function () {
 						],
 						'fromFieldName' => 'failingAuthConnection',
 						'resolve'       => static function () {
-							return [ 'nodes' => [ null, false, 0 ] ];
+							return [ 'nodes' => [ '', false, 0 ] ];
```

Comment:
why did this change? Is there a regression somewhere we need to document?

Reply:
If you return a `null` for an object's root you GraphQL does not resolve that types fields. Which was making this test fail.

Reply:
Specifically, the Connection Interfaces release made `nodes` non nullable. The error was just getting suppressed before.

Reply:
Verbose pre-change error log in this workflow attempt: https://github.com/wp-graphql/wp-graphql/actions/runs/8918725858/job/24495559399

Reply:
Okay, I think I figure out the way the test was suppose to be in the first place and fixed it.

Reply:
> Specifically, the Connection Interfaces release made `nodes` non nullable. The error was just getting suppressed before.

That first node was never supposed to be a falsy value in the first place. I change it to `test`.